### PR TITLE
feat(ext/url): URLSearchParams.size

### LINKED ISSUE #17882: URL: add URLSearchParams's size
Spec: https://github.com/whatwg/url/pull/734

Tests: https://github.com/web-platform-tests/wpt/pull/38655

### PR DESCRIPTION
**Pull request by lino-levan**
Closes #17882.

### CODE REVIEW

File: ext/url/00_url.js [hunk 1]
```diff
@@ -302,6 +302,10 @@ class URLSearchParams {
     webidl.assertBranded(this, URLSearchParamsPrototype);
     return ops.op_url_stringify_search_params(this[_list]);
   }
+
+  get size() {
```
Comment (crowlKats) [inline ext/url/00_url.js:306]: this needs a webid brand assertion

Comment (lino-levan) [reply ext/url/00_url.js:306]: Of course, how could I be so foolish.

**Comment by lino-levan** [comment]
Looks like the WPT tests landed, will enable them.

**Comment by lino-levan** [comment]
@crowlKats pls take a look

**Review by crowlKats — APPROVED** [review]
LGTM

**Review by aapoalas — APPROVED** [review]
LGTM. Some hopefully unrelated test failure as I cannot understand how this would cause the following error:

> Uncaught (in worker "") TypeError: globalThis_.addEventListener is not a function

**Comment by andreubotella** [comment]
> LGTM. Some hopefully unrelated test failure as I cannot understand how this would cause the following error:
> 
> > Uncaught (in worker "") TypeError: globalThis_.addEventListener is not a function

This is #17405, a known flaky test.

**Comment by bartlomieju** [comment]
MDN compat tables should be updated to reflect that this API will be available in v1.32: https://github.com/mdn/browser-compat-data

**Comment by lino-levan** [comment]
Will do.

**Pull request merged**